module_spam.lua

```diff
@@ -0,0 +1,120 @@
+-- Copyright (C) 2021 Julien Castiaux
+-- This file is part of the "Not a Bot" application
+-- For conditions of distribution and use, see copyright notice in LICENSE
+
+local client = Client
+local discordia = Discordia
+local bot = Bot
+local enums = discordia.enums
+local moduleModo = require('module_modo.lua')
+
+Module.Name = "spam"
+scoreThreshold = 3
+
+
+function Module:GetConfigTable()
+	return {
+		{
+			Name = "ShouldAlert",
+			Description = "Alert moderators on spam detection (requires module_modo)",
+			Type = bot.ConfigType.Boolean,
+			Default = true
+		},
+		{
+			Name = "ShouldMute",
+			Description = "Mute spammer on spam detection (requires module_modo)",
+			Type = bot.ConfigType.Boolean,
+			Default = true
+		},
+		{
+			Name = "ShouldDelete",
+			Description = "Delete message on spam detection",
+			Type = bot.ConfigType.Boolean,
+			Default = true
+		},
+	}
+end
+
+function computeSpamScore(message)
+	local lowerContent = message.content:lower()
+	local spamScore = 0
+
+	local spamWords = {"nitro", "discord", "steam", "free"}
```

Comment:
`spamWords` and `spamHints` should be configurable. And the following values could be good defaults.